### PR TITLE
fix(): remove button from input's outer container

### LINKED ISSUE #617: Unable to populate choice when create a proposal on mobile
On mobile i'm not able to populate choices field when creating on a proposal. When i click on the input nothing happen. This should be fixed.

### PR DESCRIPTION
**Pull request by harshmaur**
https://github.com/snapshot-labs/snapshot/issues/617

This fixes the issue on mobiles where the input is inaccessible.

Fixes #617

### CODE REVIEW

**Comment by samuveth** [comment]
@harshmaur looks good! Some things that I found:
- the input cursor should be centered
- the input number color should be `text-skin-link`
image
- we could use inline tailwind classes for all this styling 
image

**Comment by harshmaur** [comment]
@samuveth I could have centered it, but I thought it didn't look so good being centered. If we need it centered, I'll do that. 

Re. the css, I would have preferred to use tailwind but I saw the original button using scss so I thought that was the convention we are following.

**Comment by samuveth** [comment]
> @samuveth I could have centered it, but I thought it didn't look so good being centered. If we need it centered, I'll do that.
> 
> Re. the css, I would have preferred to use tailwind but I saw the original button using scss so I thought that was the convention we are following.

I think we discussed all this already in DM. Let me know if there is anything else!

**Comment by harshmaur** [comment]
@samuveth changes done. Could not convert the css to tailwind because it has an important tag and that is not available in tailwind. 

I believe the !important from .border class should be removed.

**Comment by samuveth** [comment]
> @samuveth changes done. Could not convert the css to tailwind because it has an important tag and that is not available in tailwind.
> 
> I believe the !important from .border class should be removed.

You can try using the tailwind class with a `!` like this f.e `!border-XX`

File: src/components/Ui/Input.vue [hunk 1]
```diff
@@ -23,8 +25,22 @@ function handleInput(e) {
 </script>
 
 <template>
-  <UiButton
-    class="text-left w-full mb-2 flex px-3"
+  <div
+    class="
+      border border-skin-border
```
Comment (samuveth) [inline src/components/Ui/Input.vue:30]: Maybe we should we move this into a class f.e `fake-button` inside `style.css` so we can reuse in other components.

**Comment by bonustrack** [comment]
@harshmaur Some input in settings are broken now (too much height)
image

**Review by samuveth — APPROVED** [review]
Looks good!

**Pull request merged**